core/src/main/scala/cats/Parallel.scala

```diff
@@ -106,6 +106,14 @@ trait Parallel[M[_]] extends NonEmptyParallel[M] {
 
       override def whenA[A](cond: Boolean)(f: => F[A]): F[Unit] = applicative.whenA(cond)(f)
     }
+
+  /**
+   * Like [[Applicative.replicateA]], but uses the apply instance
+   * corresponding to the Parallel instance instead.
+   */
+  def parReplicateA[A](n: Int, ma: M[A]): M[List[A]] =
+    if (n <= 0) monad.pure(List.empty[A])
+    else Parallel.parSequence(List.fill(n)(ma))(UnorderedFoldable.catsTraverseForList, this)
```

Comment:
To me it seems that `trait Parallel` is designed to just represent a relationship between a Monad and Applicative with parallel composition. But most of complimentary methods are placed into `object Parallel` (including `parAp` per se). I'm not absolutely confident, but shouldn't `parReplicateA` be also placed in the object rather than the trait?